Code/Framework/AzFramework/Platform/Windows/AzFramework/Application/Application_Windows.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzFramework\Application\Application.h>
+#include <AzFramework\API\ApplicationAPI_Windows.h>
+#include <AzCore\Memory\SystemAllocator.h>
```

Comment:
This must be changed to forward slashes
```suggestion
#include <AzFramework/Application/Application.h>
#include <AzFramework/API/ApplicationAPI_Windows.h>
#include <AzCore/Memory/SystemAllocator.h>
```